src/carousel/carousel.scss

```diff
@@ -12,17 +12,29 @@
 
 @mixin iui-carousel-slider {
   display: flex;
+  gap: $iui-s;
```

Comment:
What's the reason for the gap between slides?

Reply:
Yeah, question ,if we want to show empty space between every slide.. It is visible when sliding...

Reply:
it's for this third case mentioned in description
> allow showing multiple slides

![image](https://user-images.githubusercontent.com/9084735/156773571-cda0dc3b-3066-4ac2-969f-175939ca52fa.png)

Reply:
Ahhh, ok, I understand now.  That will be a nice addition.